libs/blocks/marquee/mnemonic-list.js

```diff
@@ -0,0 +1,26 @@
+import { getConfig, createTag, loadStyle } from '../../utils/utils.js';
+
+export const decorateMnemonicList = (container) => {
+  if (!container) return;
+  const paragraphs = container.querySelectorAll(':scope p:not([class])');
+  if (paragraphs.length < 1) return;
+  const { miloLibs, codeRoot } = getConfig();
+  loadStyle(`${miloLibs || codeRoot}/blocks/marquee/mnemonic-list.css`);
+  const productList = createTag('div', { class: 'product-list' });
+  [...paragraphs].forEach((paragraph) => {
+    const title = paragraph.querySelector('strong');
+    const picture = paragraph.querySelector('picture');
+    const product = createTag('div', { class: 'product-item' });
+    if (title && !picture) {
+      product.appendChild(title);
+    }
+    if (title && picture) {
+      product.appendChild(picture);
+      product.appendChild(title);
+    }
```

Comment:
Is this repetition needed for some use-cases? Otherwise we might get away with just
```
if (picture) product.appendChild(picture);
if (title) product.appendChild(title);
```

Reply:
Now that you mention it, there is no need for the repetition so I removed it.